vectara_agentic/tools.py

```diff
@@ -182,7 +182,7 @@ def rag_function(*args, **kwargs) -> dict[str, Any]:
                 summary_num_results=summary_num_results,
                 summary_response_lang=summary_response_lang,
                 summary_prompt_name=vectara_summarizer,
-                vectara_query_mode=reranker,
+                reranker=reranker,
```

Comment:
This is an important bugfix: the previous version did not use the right reranker